packages/tocco-ui/src/EditableValue/EditableValue.stories.js

```diff
@@ -0,0 +1,165 @@
+/* eslint-disable no-console */
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {withKnobs, number, text, boolean} from '@storybook/addon-knobs'
+import {IntlProvider} from 'react-intl'
+
+import EditableValue from './'
+import {htmlMarkup} from '../util/htmlMarkup'
+import {SelectStory} from '../Select/Select.stories'
+import {UploadStory} from '../Upload/Upload.stories'
+import {Upload as Raw} from '../Upload/Upload'
+
+const EMPTY_FUNC = () => {}
+
+storiesOf('Edit Data/EditableValue', module)
```

Comment:
its hard to test the components if you have to edit the value with knobs. This does not simulate the behavior. Can you change that?